python/paddle/v2/fluid/io.py

```diff
@@ -60,7 +60,12 @@ def _clone_var_in_block_(block, var):
         persistable=True)
 
 
-def save_vars(executor, dirname, main_program=None, vars=None, predicate=None):
+def save_vars(executor,
+              dirname,
+              main_program=None,
+              vars=None,
+              predicate=None,
+              save_file_name='__parameters__'):
```

Comment:
I suggest setting the default value of `save_filename` to `None`. If `None`, then all variables will be saved into separate files as before. I am not sure if it is suitable to change the storing format of training results. So maybe it is better to enable two formats?

Reply:
Done